src/Adapter/Currency/CurrencyDataProvider.php

```diff
@@ -80,9 +80,9 @@ public function findAll()
      *
      * @param string $isoCode
      *                        An ISO 4217 currency code
-     * @param int|null $idLang
-     *                         Set this parameter if you want the currency in a specific language.
-     *                         If null, default language will be used
+     * @param int|false|null $idLang
+     *                               Set this parameter if you want the currency in a specific language.
+     *                               If null, default language will be used
```

Comment:
```suggestion
     *                               If null or false, default language will be used
```
Does it make sense ?